src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -156,7 +156,99 @@ mocha.describe('readings API', () => {
 
 				// Add C9 here
 
-				// Add C10 here
+				mocha.it('C10: 1 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as BTU', async () => {
+					// Use predefined unit and conversion data
+					const unitData = unitDatakWh.concat([
+						//adding u1, u2, u3, u16
+						{ 	//u1
```

Comment:
OED prefers not to have comments at the end of the line. Could you move these (this one will be removed but for later ones) to the next line above the code being described.